README.md

```diff
@@ -118,6 +118,23 @@ sentry_set_commits(
 )
 ```
 
+#### Create deploy
+
+Creates a new release deployment for a project on Sentry.
+
+```ruby
+sentry_create_deploy(
```

Comment:
`h`: I think we should also add the following optional parameters as we have it for `sentry_upload_dsym`. Of course not only here, but also in the action 😁.
```
api_key: '...',
auth_token: '...',
org_slug: '...',
project_slug: '...',
```

Reply:
I added them in `README`, in the actions, this is already handled through `Helper::SentryConfig.parse_api_params(params)`